frontend/server/src/DAO/QualityNominations.php

```diff
@@ -336,8 +338,28 @@ function (string $type): string {
             $params[] = $nominatorUserId;
         }
 
+        if (!is_null($query)) {
+            /*Because renamed columns in the query*/
```

Comment:
nit:

```suggestion
            // Some columns are renamed in the query.
```

Los comentarios del estilo `/*...*/` hay que evitarlos siempre que se pueda en favor de los `//`.

Reply:
Si listo